### ISSUE TITLE
EQU Ver 5.0

### ISSUE THREAD

**Issue by joannplano2005** [VCCE/VCC#58]
EQU Ver 5 loads and runs. Problem is VCC is Write Protecting it. Can not make directories. get Error 242 - Write Protect. Same if I try and save a text file. Drive 0 and Drive 1 also Write Protected.

**Comment by BGPierce** [comment]
First, VCC has NO write protection scheme on any drive, it's not even in the program code. If any "Write Protected" errors show, then Windows has write protected it, not VCC.

I just loaded EoU Beta 5 into VCC:

FD-502 Drive 0 = 63EMU.DSK
HD Drive 0 = 63SDC.VHD

I typed DOS at the RSDOS cmd prompt, NitrOS9 boots.
I created a disk in "/D1" and formatted it.
I typed "makdir /d1/CMDS"
I type "dir /d1" and "CMDS" was there
I made several more directories as well as subdirectories on "/D1", no problem.
I batch copied a group of files to "/D1/CMDS" successfully.
I repeated the above on "/D0".
I typed "makdir /DD/TEMP" successfully
I batch copied a group of files into "/DD/TEMP"
I typed "deldir /DD/TEMP" and directory was successfully deleted.

I cannot duplicate your errors as you describe them.
Suggestion, check you individual disk files and see if they are write protected by Windows (or the containing folder itself).
Also, are these disk images being stored under "Program Files" or "Program Files (x86)"? If so, those directories are write protected by Windows 7-10 and do not allow you to alter the contents in those directories.
Try running VCC "as Administrator" and see if that solves the problem. If so, then Windows is your culprit.

Bill P.

**Comment by ejaquay** [comment]
When Vcc mounts a read only dsk or vhd file it marks it as write protected.  Disks reads will still work but writes will report write protect error.  This so users can set the dsk files on windows read only to get the write protect effect.

**Comment by BGPierce** [comment]
JoAnn, there's your answer above. Your Windows files are marked read-only, therefore VCC sees them as write protected. I personally did not know this feature was in VCC and I've used it since version 1.0 when Joseph was beta testing it.
So, right click your disk file, uncheck "read only" and the problem is solved.
Thanks for the info EJ.
Bill P.

**Comment by joannplano2005** [comment]
Funny that VCC is NOT in Program Files or Program Files (x86) but MAME64 IS under Program Files. With MAME64 I CAN make directories and write files. So if Program Files is Write Protected them why does the MAME64 version work I made sure that the same files are used in both emulators.

ED,
OK changed the permission to Full Rights on both files and now can make directories and write files.
JoAnn Donaldson
    On Tuesday, December 15, 2020, 12:25:26 PM CST, Bill Pierce <notifications@github.com> wrote:  
 
 


JoAnn, there's your answer above. Your Windows files are marked read-only, therefore VCC sees them as write protected. I personally did not know this feature was in VCC and I've used it since version 1.0 when Joseph was beta testing it.
So, right click your disk file, uncheck "read only" and the problem is solved.
Thanks for the info EJ.
Bill P.
